### ISSUE TITLE
bug(material/chips): single-select chips visually communicate selected state with color alone (WCAG 1.4.1)

### ISSUE THREAD

**Issue by zarend** [angular/components#25886]
### Description

single-select chips visually communicate selected state with color alone. It should be visually communicated with more than just color.

https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-without-color.html

### Reproduction

Steps to reproduce:
1. open https://material2-dev.web.app/mdc-chips
2. Navigate to "Selectable Chips" demo
3. Under "Single selection", click "Medium"

### Expected Behavior

selected state of "Medium" is visually communicated with more than color alone

### Actual Behavior

color of "Medium" changes to indicate selected state

<img width="592" alt="image" src="https://user-images.githubusercontent.com/7720245/198362096-688f5a37-67ad-4f7e-9f40-7446ac29c41c.png">

**Comment by zarend** [comment]
#25890 fixed this issue
